practices/managing-node-projects.md

```diff
@@ -10,7 +12,30 @@
 * [Dependency management tools](#dependency-management-tools)
 * [Handling dependencies from third parties](#handling-dependencies-from-third-parties)
 
-The following guide describes how we manage package dependencies in [Springer Nature Node.js-based projects](https://github.com/springernature?utf8=%E2%9C%93&q=&type=public&language=javascript).
+The following guide describes how we use node and manage package dependencies in [Springer Nature Node.js-based projects](https://github.com/springernature?utf8=%E2%9C%93&q=&type=public&language=javascript).
+
+
+## Specifying versions of node
+
+It is important to specify which versions of node your application expects. There are two ways of doing this, and you are encouraged to do both.
+
+1. The [`engines`](https://docs.npmjs.com/files/package.json#engines) field in `package.json`:
+    * Important if authoring libraries, because `npm install` only _enforces_ compatibility on the basis of the `engines` field if the package is a dependency.
```

Comment:
I'm having some trouble parsing this sentence - what's the emphasis on _enforces_ for? Only enforces as opposed to what? Just giving a warning, or...?

Might help to split the sentence in two, with the first sentence being a firm directive to use `engines` if you're writing a library, and the 2nd explaining why